### ISSUE TITLE
Scroll down instead of up when xreader regains focus

### ISSUE THREAD

**Issue by cirnod** [linuxmint/xreader#325]
```
 * Xreader version: 2.0.2
 * Distribution: Linux Mint 19.1
```

**Issue**
If xreader regains focus and is in "Page view" mode, it switches to the wrong page (next page instead of previous page) when performing a scroll up action. 

**Steps to reproduce**
Open any pdf in "Page view" mode, select any page but the first one, focus any other application than xreader, re-focus on xreader, perform a single scroll up action. 

**Expected behaviour**
Previous page should be displayed. 

**Other information**
NAME="Linux Mint"
VERSION="19.1 (Tessa)"
ID=linuxmint
ID_LIKE=ubuntu
PRETTY_NAME="Linux Mint 19.1"

xreader version: 2.0.2

I tried to compile and install the latest (2.2.2.) version of xreader but it failed when I tried to start xreader with 
```
/usr/local/bin/xreader: symbol lookup error: /usr/local/bin/xreader: undefined symbol: ev_document_misc_get_screen_dpi_at_window
```

**Comment by ssepan2** [comment]
I second this: when switching back and using the mouse wheel to scroll up, the first detent goes down one page. Thereafter, each detent in the up direction goes up as expected. 

xreader v2.2.3

[https://termbin.com/m78j]

**Comment by icarter09** [comment]
Xreader v2.6.4
Mint 20 64bit.

I'm not able to duplicate this issue.

@ssepan2 @cirnod  can either of you see if you are still experiencing this issue? Thanks.

**Comment by ssepan2** [comment]
> Xreader v2.6.4
> Mint 20 64bit.
> 
> I'm not able to duplicate this issue.
> 
> @ssepan2 @cirnod can either of you see if you are still experiencing this issue? Thanks.

@icarter09, Just re-tried it (I'm on 2.6.4 now too, instead of 2.2.3 when I last checked.): I'm not seeing it now. I AM seeing that the first detent up OR down does nothing, THEN begins in the right direction on 2nd detent and thereafter.

I am on Mint 20 64bit also

**Comment by icarter09** [comment]
@icarter09 I am seeing that as well.

**Comment by cirnod** [comment]
I also retried with xreader 2.6.4 and Linux Mint 20 Cinnamon: With my setup scrolling in _Page View_ mode has no longer any effect (neither for the first nor any following detent, no matter whether I focus any other application in between). However, it is still possible to switch pages by using the "Go to the next page"/"Go to the previous page" buttons and by using the arrow keys of the keyboard. Using the scroll wheel in _Reader View_ works as expected.

**Comment by ssepan2** [comment]
@cirnod , I noticed that too: When View|Continuous is _not_ selected, the scroll wheel will not scroll the document pages; it does work for me when Continuous _is_ selected.

**Comment by cirnod** [comment]
@ssepan2 I agree, when I select View|Continuous after having selected _Page View_ the scroll wheel has an effect but basically it behaves like in _Reader View_, i.e. only part of the page is scrolled, not entire pages as this was the case before (e.g. in version 2.0.2). Also whenever I select View|Continuous both of the 2 view option buttons (_Page View_ and  _Reader View_) get deselected. When I select a view option again, View|Continuous is deselected again. Based on these observations, I would consider it as a bug, too.
